requirements.txt

```diff
@@ -9,7 +9,7 @@ tensorboard
 onnx
 glog
 torchvision
-torch
+torch==1.7.1
```

Comment:
Why do you need exactly this version?

Reply:
I ran into a problem on the cluster when leaving the torch w/o version. So, I hardcoded a version of what I have locally.

Reply:
It is also a good reason to highlight the version explicitly to not face the problems with different versions of the PyTorch

Reply:
If it's better to work with 1.8 PyTorch, I will try to migrate to 1.8 in future PR

Reply:
I've launched training on cluster with torch==1.8.1, will see how it works